test/commands/overlay.test.ts

```diff
@@ -0,0 +1,31 @@
+import base, { expect } from '@oclif/test';
+
+const test = base;
+
+describe('overlay subcommand', () => {
```

Comment:
I think we're missing some tests here, of the applyOverlay method. I know it's some copy/paste that "should work", but it would probably help to add some unit tests to ensure that if we refactor or enhance it, we don't break anything. WDYT?

Reply:
Is this something you will do in this PR or in another one?

Reply:
I've added unit tests to the `definition.ts` class. And added a new test case for the `--out` flag of the `overlay` command. It should cover most cases.